test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Converters/IconUrlToImageCacheConverterTests.cs

```diff
@@ -89,6 +89,24 @@ public void Convert_WithLocalPath_LoadsImage()
             Assert.Equal(iconUrl, image.UriSource);
         }
 
+        [Fact]
+        public void Convert_WithLocalPathAndColorProfile_LoadsImage()
+        {
+            var iconUrl = new Uri(@"resources/grayicc.png", UriKind.Relative);
```

Comment:
Normally, IconUrlToImageCacheConverter accepts either embedded icon URIs (an absolute URI to a package filepath with a fragment part, pointing to the relative icon path inside the package) or absolute URIs. Since we are just testing that image is read, it's fine to leave the relative URI.

@rrelyea